rfcs/formats/identity.md

```diff
@@ -0,0 +1,203 @@
+
+A permissioned blockchain network has a Network DID that represents it.
+The Network DID document conforms to the [did-core](https://www.w3.org/TR/did-core) specifications.
+
+**id** - This property is mandatory which is the DID URI for the network. This id (DID) can be used to resolve the DID Document of the network from the IIN DID registry.
+
+**verificationMethod** - One verification method property must be present in the Network DID document and it must be able to support [Group Control](https://www.w3.org/TR/did-core/#group-control), so that the network participant units can jointly control the Network DID Document. Additionally, the verification method must specify who can update the DID in case the network changes as participant units leave and new ones join. We describe a multi-signature based verificationMethod type in the following sections.
+
+For interoperation in the data plane, network specific identity credentials such as certificates (X.509 certificates in Hyperledger Fabric networks) are required to be configured.  For these data plane credentials, there can be optional verificationMethods in the Network DID document. For better privacy, wothout directly expressing the data plane credentials within the Network DID document, only their cryptographic hash may be included which will ensure no rogue network participant can produce tampered data plane credentials deny/attack the interoperation. 
+
+<!-- These credentials can be expressed as a verificationMethod in the Network DID Document itself, but there might be privacy concerns regarding it since the actual identity of the participant units might be revealed directly in the public ledger through this. -->
+
+**authentication** - The authentication verification relationship must point to the group controlled verification method of the Network DID.
+
+**networkParticipants** - This is not a core property of the DID specification. This is a special property required for Network DIDs that specify the members in the network. It is a list of DIDs of the individual members.
+
+### Multi-signature based verificationMethod
+
+A network DID must be controlled jointly by the network's participant units for operations such as creation of the DID, and its updation with changing network structure, and for other operations such as rotating keys, updating verification methods etc.. To enable this group control we introduce a verificationMethod that uses multi-signatures and policies.
```

Comment:
"methods, etc." (single period).
"control, we".
"verificationMethod" in code font.
"and associated policies".
"rotating/refreshing keys"?